_hooknode/Controller/HookNode.php

```diff
@@ -0,0 +1,62 @@
+<?php
+namespace Controller;
+
```

Comment:
@awmpietro   All the code looks good (although I'm not really familiar with databases) but I think some of these files are in the wrong repository.  

The files that are involved in the following:
-storing or creating a DB of hook nodes
-adding a hook node to the db
-selecting a hood node
-modifying a hook node score
-modifying a hook node status 

...All would need to be in the brokernode repository.  

Basically, the brokernode needs to have a DB of hook nodes and the methods to interact with that DB.  

The hooknode needs to have a DB of broker node and the methods to interact with that DB.

Reply:
Ok, Thanks @rfornea. I'll change it

Reply:
Submitted to the other repo. Review please, i will cancel this PR